### DISCUSSION TITLE
BottomSheet background color when dragging up

### DISCUSSION THREAD

**Discussion by drochetti** [gorhom/react-native-bottom-sheet#386] [Q&A] [answered]
Hi community, I'm seeking help to support light and dark themes in my app and have the BottomSheet act accordingly.

When on dark mode, the underlying container of the BottomSheet is white when dragging the BottomSheet up beyond its limits (when it reaches the friction point) appears and it's white (see attached image).

I tried setting the `backgroundColor` of `<BottomSheetModal>`  with no luck. Did I miss something?

I'm using:

- `react-native@0.64.0`
- `@gorhom/bottom-sheet@3.6.2`
- `react-native-reanimated@2.0.1`

---

Screenshot:

![bottom-sheet-drag-up-spring-effect-background-annotated](https://user-images.githubusercontent.com/319413/113491572-c6fe1180-9486-11eb-8dd6-29b03cbae5fd.png)

**Comment by gorhom** [comment, marked as answer]
try to set `backgroundComponent` to `nulll`
